foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/service/ConfigService.java

```diff
@@ -32,14 +36,18 @@ public class ConfigService {
     @Inject
     public ConfigLoader configLoader;
 
+    @Inject
+    public ConfigStoreInit configStoreInit;
```

Comment:
A: I would prefer that ConfigStoreInit be independent from this class since it fetches its own configLoader and are we going to handle force loading in the loadProperties() function

Reply:
If we make these classes independent, then we have to duplicate the logic that currently lives in `ConfigStoreInit.init()` and maintain it in two places, so was trying to avoid that. However if we envision the `ConfigService` being slightly different, which I think we do with the force reloading, then I think it makes sense to duplicate that logic for now with the understanding that it will eventually be different.

Reply:
We should just drop the `/load` endpoint IMO. It wasn't in our original feature roadmap, and it seems more like a security concern than anything.

Reply:
Yeah the function is not required but kept based on the idea that user can have a way for user to reload the property. If it becomes a overhead and that's not a function that we are interested to support reload function, which I think we can do it by restart the service. Let's drop it to keep thing simple.

Reply:
Removed